### PR TITLE
fix(designer): Typo and text formatting

### PR DESCRIPTION
**Pull request by ccastrotrejo**
### Main Code Changes
- Update in typos and text formatting

### CODE REVIEW

**Review by refortie — APPROVED** [review]

File: Localize/lang/strings.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
   "+JCU+Y": "Select this if you are configuring an on-prem connection",
   "+W2vTp": "Please enter an e-mail address",
```
Comment (ecfan) [inline Localize/lang/strings.json:8]: - "please" isn't used appropriately here. See ["please" - Microsoft Style Guide](https://learn.microsoft.com/style-guide/a-z-word-list-term-collections/p/please).
- "email" doesn't use a hyphen. See ["email" - Microsoft Style Guide](https://learn.microsoft.com/style-guide/a-z-word-list-term-collections/e/email):

```suggestion
  "+W2vTp": "Enter an email address.",
```

File: Localize/lang/strings.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
   "+JCU+Y": "Select this if you are configuring an on-prem connection",
```
Comment (ecfan) [inline Localize/lang/strings.json:7]: What does 'this" reference? Use "on-premises", not "on-prem". See ["on-premises" - Microsoft Style Guide](https://learn.microsoft.com/style-guide/a-z-word-list-term-collections/o/off-premises-on-premises).

```suggestion
  "+JCU+Y": "Select this if you are configuring an on-premises connection.",
```

File: Localize/lang/strings.json [hunk 3]
```diff
@@ -7,6 +7,7 @@
...
   "+W2vTp": "Please enter an e-mail address",
   "+W9fkq": "(UTC+08:00) Kuala Lumpur, Singapore",
+  "+ZSBrq": "Context menu for {title} card",
   "+dlOgj": "(UTC-05:00) Haiti",
   "+g+mch": "Returns the body for a part in a multipart output of an action",
```
Comment (ecfan) [inline Localize/lang/strings.json:12]:
```suggestion
  "+g+mch": "Returns the body for a part in a multipart output of an action.",
```

File: Localize/lang/strings.json [hunk 4]
```diff
@@ -7,6 +7,7 @@
...
   "+iMTt7": "Check to select this connection",
```
Comment (ecfan) [inline Localize/lang/strings.json:13]: - Use "select", not "check". See ["check" - Microsoft Style Guide](https://learn.microsoft.com/style-guide/a-z-word-list-term-collections/c/check-checkbox-check-mark).

```suggestion
  "+iMTt7": "Select the checkbox to choose this connection.",
```

File: Localize/lang/strings.json [hunk 5]
```diff
@@ -266,7 +266,6 @@
   "AO6T9u": "Done",
   "AX7dq3": "Input node type does not match the schema node's type",
```
Comment (ecfan) [inline Localize/lang/strings.json:267]:
```suggestion
  "AX7dq3": "The input node type doesn't match the schema node's type.",
```

File: Localize/lang/strings.json [hunk 6]
```diff
@@ -266,7 +266,6 @@
...
   "AYaqW/": "(UTC+06:30) Yangon (Rangoon)",
-  "AbZRJ7": "select all text in {label}",
   "AcdX+2": "Insert data from previous step (You can also add by typing / in the editor)",
```
Comment (ecfan) [inline Localize/lang/strings.json:269]:
```suggestion
  "AcdX+2": "Enter the data from previous step. You can also add data by typing the '/' character.",
```

File: Localize/lang/strings.json [hunk 7]
```diff
@@ -266,7 +266,6 @@
...
   "Aci186": "{label} Add dynamic data pressing Alt + /",
```
Comment (ecfan) [inline Localize/lang/strings.json:270]:
```suggestion
  "Aci186": "{label} To add dynamic data, press Alt + /",
```

File: Localize/lang/strings.json [hunk 8]
```diff
@@ -266,7 +266,6 @@
...
   "AgrMqq": "Type of '{0}' is '{1}'.",
```
Comment (ecfan) [inline Localize/lang/strings.json:271]:
```suggestion
  "AgrMqq": "The type for '{0}' is '{1}'.",
```

File: Localize/lang/strings.json [hunk 9]
```diff
@@ -585,7 +583,6 @@
   "QEp3C8": "Convert the input to a JSON type value",
   "QGbUXX": "Status code",
   "QIOOQU": "No Items",
-  "QQYzHC": "Context menu for {title} card",
   "QZ5DZl": "alt text for chevron down",
```
Comment (ecfan) [inline Localize/lang/strings.json:582]:
```suggestion
  "QZ5DZl": "Alt text for down chevron",
```

File: Localize/lang/strings.json [hunk 10]
```diff
@@ -585,7 +583,6 @@
   "QEp3C8": "Convert the input to a JSON type value",
   "QGbUXX": "Status code",
   "QIOOQU": "No Items",
```
Comment (ecfan) [inline Localize/lang/strings.json:581]:
```suggestion
  "QIOOQU": "No items",
```

File: Localize/lang/strings.json [hunk 11]
```diff
@@ -585,7 +583,6 @@
   "QEp3C8": "Convert the input to a JSON type value",
```
Comment (ecfan) [inline Localize/lang/strings.json:579]:
```suggestion
  "QEp3C8": "Converts the input to a JSON type value.",
```

File: Localize/lang/strings.json [hunk 12]
```diff
@@ -700,6 +697,7 @@
   "W070M2": "of {max}",
   "W6FdMh": "Required. The name of the new property.",
   "WGwH45": "Clear",
+  "WK6hZX": "Select all text in {label}",
   "WLj8DH": "Please enter a valid dictionary",
```
Comment (ecfan) [inline Localize/lang/strings.json:698]: "Please" isn't necessary or appropriate:

```suggestion
  "WLj8DH": "Enter a valid dictionary.",
```

File: Localize/lang/strings.json [hunk 13]
```diff
@@ -700,6 +697,7 @@
...
   "WOOYW1": "Actions perform operations on data or communicate between systems to execute.",
```
Comment (ecfan) [inline Localize/lang/strings.json:699]: "systems to execute"?

```suggestion
  "WOOYW1": "Actions perform operations on data, communicate between systems, or run other tasks.",
```

File: Localize/lang/strings.json [hunk 14]
```diff
@@ -782,6 +780,7 @@
   "_+JCU+Y.comment": "Tooltip for on-prem gateway connection checkbox",
   "_+W2vTp.comment": "Placeholder description for a newly inserted Email parameter",
   "_+W9fkq.comment": "Timezone value ",
```
Comment (ecfan) [inline Localize/lang/strings.json:777]: Should trailing whitespace exist here?

```suggestion
  "_+W9fkq.comment": "Timezone value",
```

File: Localize/lang/strings.json [hunk 15]
```diff
@@ -782,6 +780,7 @@
   "_+JCU+Y.comment": "Tooltip for on-prem gateway connection checkbox",
   "_+W2vTp.comment": "Placeholder description for a newly inserted Email parameter",
```
Comment (ecfan) [inline Localize/lang/strings.json]:
```suggestion
  "_+W2vTp.comment": "The placeholder description for a newly inserted 'Email' parameter.",
```

File: Localize/lang/strings.json [hunk 16]
```diff
@@ -782,6 +780,7 @@
   "_+JCU+Y.comment": "Tooltip for on-prem gateway connection checkbox",
```
Comment (ecfan) [inline Localize/lang/strings.json]: "on-premises", not "on-prem":

```suggestion
  "_+JCU+Y.comment": "Tooltip for the on-premises data gateway connection checkbox",
```

File: Localize/lang/strings.json [hunk 17]
```diff
@@ -782,6 +780,7 @@
...
   "_+W2vTp.comment": "Placeholder description for a newly inserted Email parameter",
   "_+W9fkq.comment": "Timezone value ",
+  "_+ZSBrq.comment": "Accessibility label",
   "_+dlOgj.comment": "Timezone value ",
```
Comment (ecfan) [inline Localize/lang/strings.json:779]: Trailing whitespace?

```suggestion
  "_+dlOgj.comment": "Timezone value",
```

File: Localize/lang/strings.json [hunk 18]
```diff
@@ -782,6 +780,7 @@
...
   "_+g+mch.comment": "Label for description of custom multipartBody Function",
```
Comment (ecfan) [inline Localize/lang/strings.json]:
```suggestion
  "_+g+mch.comment": "Label for the description of the custom 'multipartBody' function",
```

File: Localize/lang/strings.json [hunk 19]
```diff
@@ -782,6 +780,7 @@
...
   "_+iMTt7.comment": "aria label description for check button",
```
Comment (ecfan) [inline Localize/lang/strings.json]: What is an "aria label"? What's a "check button"?

```suggestion
  "_+iMTt7.comment": "Aria label description for check button",
```

File: Localize/lang/strings.json [hunk 20]
```diff
@@ -4,12 +4,9 @@
...
   "+3efoo": "No schema is added",
   "+DxBpE": "Failed loading the schema. Please try again.",
-  "+JCU+Y": "Select this if you are configuring an on-prem connection",
-  "+W2vTp": "Please enter an e-mail address",
   "+W9fkq": "(UTC+08:00) Kuala Lumpur, Singapore",
+  "+ZSBrq": "Context menu for {title} card",
   "+dlOgj": "(UTC-05:00) Haiti",
-  "+g+mch": "Returns the body for a part in a multipart output of an action",
-  "+iMTt7": "Check to select this connection",
   "+jnjtp": "Select a Manual Workflow resource",
```
Comment (ecfan) [inline Localize/lang/strings.json:10]: The original text is confusing. Do customers know what is a "manual workflow resource"? Does this really mean a workflow with a Request trigger, which is named 'manual' in code view?

```suggestion
  "+jnjtp": "Select workflow with 'manual' trigger",
```

File: Localize/lang/strings.json [hunk 21]
```diff
@@ -4,12 +4,9 @@
   "+20fdv": "Invalid properties '{0}' for authentication type '{1}'.",
   "+3efoo": "No schema is added",
```
Comment (ecfan) [inline Localize/lang/strings.json:5]:
```suggestion
  "+3efoo": "No schema is added.",
```

File: Localize/lang/strings.json [hunk 22]
```diff
@@ -4,12 +4,9 @@
   "+20fdv": "Invalid properties '{0}' for authentication type '{1}'.",
```
Comment (ecfan) [inline Localize/lang/strings.json:4]:
```suggestion
  "+20fdv": "The '{0}' properties are invalid for the '{1}' authentication type.",
```

File: Localize/lang/strings.json [hunk 23]
```diff
@@ -163,11 +160,11 @@
   "4wjJs0": "14",
   "5+MuPz": "Enter a valid double.",
```
Comment (ecfan) [inline Localize/lang/strings.json:161]:
```suggestion
  "5+MuPz": "Enter a valid Double number.",
```

File: Localize/lang/strings.json [hunk 24]
```diff
@@ -4,12 +4,9 @@
...
   "+DxBpE": "Failed loading the schema. Please try again.",
-  "+JCU+Y": "Select this if you are configuring an on-prem connection",
-  "+W2vTp": "Please enter an e-mail address",
   "+W9fkq": "(UTC+08:00) Kuala Lumpur, Singapore",
+  "+ZSBrq": "Context menu for {title} card",
   "+dlOgj": "(UTC-05:00) Haiti",
-  "+g+mch": "Returns the body for a part in a multipart output of an action",
-  "+iMTt7": "Check to select this connection",
   "+jnjtp": "Select a Manual Workflow resource",
   "+lpuMl": "Required. The value to return if the expression is 'false'.",
```
Comment (ecfan) [inline Localize/lang/strings.json:11]:
```suggestion
  "+lpuMl": "Required. The value to return if the expression is false.",
```

File: Localize/lang/strings.json (hunk 17 above)
Comment (ccastrotrejo) [reply Localize/lang/strings.json:779]: Yeah it seems like so

File: Localize/lang/strings.json (hunk 14 above)
Comment (ccastrotrejo) [reply Localize/lang/strings.json:777]: yeah I think so

File: Localize/lang/strings.json (hunk 20 above)
Comment (ecfan) [reply Localize/lang/strings.json:10]: @ccastrotrejo: Any way to get clarification about the meaning? Thanks!

File: Localize/lang/strings.json [hunk 25]
```diff
@@ -163,11 +160,11 @@
   "4wjJs0": "14",
   "5+MuPz": "Enter a valid double.",
   "5FINOY": "Workflow has parameter validation errors on the following operations: {invalidNodes}",
-  "5HGtu7": "Function removed from current location. Function exists elsewhere.",
   "5J7j9k": "Required. The XPath expression to evaluate.",
   "5MWmiO": "Line Number",
   "5OvGgn": "Body",
   "5SAQOb": "Authority",
+  "5UPOY+": "Select this if you are configuring an on-premises connection.",
```
Comment (ecfan) [inline Localize/lang/strings.json:167]: Unclear what "this" references? A checkbox?

File: Localize/lang/strings.json [hunk 26]
```diff
@@ -163,11 +160,11 @@
...
   "5aa76N": "Returns the first element in the array or string passed in",
```
Comment (ecfan) [inline Localize/lang/strings.json:168]:
```suggestion
  "5aa76N": "Returns the first element in the passed-in array or string.",
```

File: Localize/lang/strings.json (hunk 25 above)
Comment (ccastrotrejo) [reply Localize/lang/strings.json:167]: Yes, a checkbox

File: Localize/lang/strings.json [hunk 27]
```diff
@@ -163,11 +160,11 @@
   "4wjJs0": "14",
   "5+MuPz": "Enter a valid double.",
   "5FINOY": "Workflow has parameter validation errors on the following operations: {invalidNodes}",
```
Comment (ecfan) [inline Localize/lang/strings.json:162]:
```suggestion
  "5FINOY": "The workflow has parameter validation errors in the following operations: {invalidNodes}",
```

File: Localize/lang/strings.json [hunk 28]
```diff
@@ -1,17 +1,11 @@
 {
   "++ZVe/": "Testing",
   "+0yxlR": "Function display",
-  "+20fdv": "Invalid properties '{0}' for authentication type '{1}'.",
-  "+3efoo": "No schema is added",
   "+DxBpE": "Failed loading the schema. Please try again.",
```
Comment (ecfan) [inline Localize/lang/strings.json:4]:
```suggestion
  "+DxBpE": "Failed to load the schema. Please try again.",
```

File: Localize/lang/strings.json [hunk 29]
```diff
@@ -1,17 +1,11 @@
...
+  "+Uvo/p": "Alt text for down chevron",
   "+W9fkq": "(UTC+08:00) Kuala Lumpur, Singapore",
+  "+ZSBrq": "Context menu for {title} card",
   "+dlOgj": "(UTC-05:00) Haiti",
-  "+g+mch": "Returns the body for a part in a multipart output of an action",
-  "+iMTt7": "Check to select this connection",
-  "+jnjtp": "Select a Manual Workflow resource",
-  "+lpuMl": "Required. The value to return if the expression is 'false'.",
   "+xiT4Q": "Pagination",
   "+ym4nO": "Tracked Properties",
```
Comment (ecfan) [inline Localize/lang/strings.json:10]:
```suggestion
  "+ym4nO": "Tracked properties",
```

File: Localize/lang/strings.json [hunk 30]
```diff
@@ -428,8 +417,10 @@
   "IX1OiA": "Failed to retrieve dynamic outputs, outputs of this operation might not be visible in subsequent actions. Error details: {message}",
   "IXy91L": "Add a target schema",
   "IbGIq3": "Error Code",
```
Comment (ecfan) [inline Localize/lang/strings.json:419]:
```suggestion
  "IbGIq3": "Error code",
```

File: Localize/lang/strings.json [hunk 31]
```diff
@@ -428,8 +417,10 @@
   "IX1OiA": "Failed to retrieve dynamic outputs, outputs of this operation might not be visible in subsequent actions. Error details: {message}",
```
Comment (ecfan) [inline Localize/lang/strings.json:417]:
```suggestion
  "IX1OiA": "Failed to retrieve dynamic outputs. As a result, this operation's outputs might not be visible in subsequent actions. Error details: {message}",
```

File: Localize/lang/strings.json [hunk 32]
```diff
@@ -428,8 +417,10 @@
...
   "IXy91L": "Add a target schema",
   "IbGIq3": "Error Code",
+  "IdOhPY": "{label} To add dynamic data, press the Alt + '/' keys.",
   "IjEKvh": "Error executing the api ''{parameters}''.",
   "IjZFjQ": "(UTC+07:00) Barnaul, Gorno-Altaysk",
+  "Io48jC": "The '{0}' properties are invalid for the '{1}' authentication type.",
   "IooQu1": "Advanced options",
   "Ip7Iax": "Returns the index of the n-th occurrence of a value within a string (case-insensitive, invariant culture)",
   "IpVwYT": "Returns the first non-null object in the arguments passed in",
```
Comment (ecfan) [inline Localize/lang/strings.json:425]:
```suggestion
  "IpVwYT": "Returns the first non-null object in the passed-in argument values.",
```

File: Localize/lang/strings.json [hunk 33]
```diff
@@ -458,6 +449,7 @@
   "K7/DnZ": "Output",
   "K90YEU": "Returns a subset of characters from a string",
```
Comment (ecfan) [inline Localize/lang/strings.json:449]:
```suggestion
  "K90YEU": "Returns a subset of characters from a string.",
```

File: Localize/lang/strings.json [hunk 34]
```diff
@@ -489,7 +481,6 @@
   "LvLksz": "Loading outputs",
   "LvpxiA": "Gateway",
   "Lwe7tQ": "(UTC+00:00) Dublin, Edinburgh, Lisbon, London",
-  "LxPh9u": "copy value of {label} to clipboard",
   "M/F2/O": "Notes can only be added while editing the inputs of a step.",
```
Comment (ecfan) [inline Localize/lang/strings.json:483]:
```suggestion
  "M/F2/O": "You can add notes only when you edit a step's inputs.",
```

File: Localize/lang/strings.json [hunk 35]
```diff
@@ -776,17 +765,11 @@
   "ZnGRMP": "(UTC+07:00) Tomsk",
   "_++ZVe/.comment": "Title for testing section",
   "_+0yxlR.comment": "Label for the function display radio group",
-  "_+20fdv.comment": "Error message when having multiple invalid authentication properties",
-  "_+3efoo.comment": "Placeholder when no schema has been added",
   "_+DxBpE.comment": "Load schema error message",
-  "_+JCU+Y.comment": "Tooltip for on-prem gateway connection checkbox",
-  "_+W2vTp.comment": "Placeholder description for a newly inserted Email parameter",
+  "_+Uvo/p.comment": "Alt text for down chevron",
   "_+W9fkq.comment": "Timezone value ",
```
Comment (ecfan) [inline Localize/lang/strings.json:769]: Two words, not one:

```suggestion
  "_+W9fkq.comment": "Time zone value ",
```

File: Localize/lang/strings.json [hunk 36]
```diff
@@ -1868,6 +1855,8 @@
   "_q1gfIs.comment": "Text on example trigger node",
   "_q2OCEx.comment": "Required parameter for new property value in addProperty function",
   "_q2w8Sk.comment": "Label for description of custom string Function",
+  "_q4ChjK.comment": "Select workflow with 'manual' trigger",
+  "_q5TkqE.comment": "Required value parameter to return given if function is false",
   "_q87X20.comment": "Label for description of custom addSeconds Function",
   "_q8vnZQ.comment": "tool tip explaining what schema validation setting does",
   "_q9hf8i.comment": "Timezone value ",
```
Comment (ecfan) [inline Localize/lang/strings.json:1861]: Two words, not one:

```suggestion
  "_q9hf8i.comment": "Time zone value ",
```

File: Localize/lang/strings.json [hunk 37]
```diff
@@ -776,17 +765,11 @@
...
-  "_+W2vTp.comment": "Placeholder description for a newly inserted Email parameter",
+  "_+Uvo/p.comment": "Alt text for down chevron",
   "_+W9fkq.comment": "Timezone value ",
+  "_+ZSBrq.comment": "Accessibility label",
   "_+dlOgj.comment": "Timezone value ",
-  "_+g+mch.comment": "Label for description of custom multipartBody Function",
-  "_+iMTt7.comment": "aria label description for check button",
-  "_+jnjtp.comment": "Select a Manual Workflow resource",
-  "_+lpuMl.comment": "Required value parameter to return given if function is false",
   "_+xiT4Q.comment": "title for pagination setting",
```
Comment (ecfan) [inline Localize/lang/strings.json:772]:
```suggestion
  "_+xiT4Q.comment": "Title for pagination setting",
```

File: Localize/lang/strings.json [hunk 38]
```diff
@@ -776,17 +765,11 @@
...
   "_+ym4nO.comment": "Label for the Tracked Properties field in the settings section",
   "_//nQG4.comment": "Label for api selection",
```
Comment (ecfan) [inline Localize/lang/strings.json:774]: Unsure to what API selection means, a connector operation?

```suggestion
  "_//nQG4.comment": "Label for API selection",
```

File: Localize/lang/strings.json [hunk 39]
```diff
@@ -1148,6 +1125,7 @@
   "_FVavKy.comment": "Label for description of custom startOfDay Function",
```
Comment (ecfan) [inline Localize/lang/strings.json:1123]:
```suggestion
  "_FVavKy.comment": "Label for the description of a custom 'startOfDay' function",
```

File: Localize/lang/strings.json [hunk 40]
```diff
@@ -1148,6 +1125,7 @@
...
   "_FXC0Zi.comment": "Timezone value ",
   "_FXLR5M.comment": "A duration of time shown in hours",
+  "_FYtDP0.comment": "The placeholder description for a newly inserted 'Email' parameter.",
   "_Fgq3Db.comment": "Required collection parameter to apply last function on",
```
Comment (ecfan) [inline Localize/lang/strings.json:1127]:
```suggestion
  "_Fgq3Db.comment": "The required collection parameter for applying the 'last' function.",
```

File: Localize/lang/strings.json [hunk 41]
```diff
@@ -1148,6 +1125,7 @@
...
   "_Fh2HDb.comment": "Label for description of custom indexOf Function",
```
Comment (ecfan) [inline Localize/lang/strings.json:1128]:
```suggestion
  "_Fh2HDb.comment": "Label for the description of a custom 'indexOf' function",
```

File: Localize/lang/strings.json [hunk 42]
```diff
@@ -2279,6 +2272,7 @@
   "kZuYgD": "Returns true if the first argument is less than the second",
   "kc5Mui": "(UTC-02:00) Mid-Atlantic - Old",
   "keb4r1": "Returns true if two values are equal",
+  "kkMrF6": "Select this if you are configuring an on-premises connection.",
```
Comment (ecfan) [inline Localize/lang/strings.json:2275]:
```suggestion
  "kkMrF6": "Select this checkbox if you're setting up an on-premises connection.",
```

File: Localize/lang/strings.json [hunk 43]
```diff
@@ -2279,6 +2272,7 @@
   "kZuYgD": "Returns true if the first argument is less than the second",
   "kc5Mui": "(UTC-02:00) Mid-Atlantic - Old",
   "keb4r1": "Returns true if two values are equal",
```
Comment (ecfan) [inline Localize/lang/strings.json:2274]:
```suggestion
  "keb4r1": "Returns true if two values are equal.",
```

File: Localize/lang/strings.json [hunk 44]
```diff
@@ -2279,6 +2272,7 @@
   "kZuYgD": "Returns true if the first argument is less than the second",
```
Comment (ecfan) [inline Localize/lang/strings.json:2272]:
```suggestion
  "kZuYgD": "Returns true if the first argument is less than the second.",
```

File: Localize/lang/strings.json [hunk 45]
```diff
@@ -2199,6 +2191,7 @@
   "gMhc5o": "Required. The string to combine into a single string.",
   "gQt/0f": "Conversion functions",
   "gRUmiA": "Info",
+  "gRyFIu": "Copy value of {label} to clipboard",
   "gTXyUo": "Time Zone",
```
Comment (ecfan) [inline Localize/lang/strings.json:2195]:
```suggestion
  "gTXyUo": "Time zone",
```

File: Localize/lang/strings.json [hunk 46]
```diff
@@ -2199,6 +2191,7 @@
...
   "gUp23R": "The name already exists or is invalid, please update it before proceeding.",
   "gWRF8m": "Trigger Conditions",
```
Comment (ecfan) [inline Localize/lang/strings.json:2197]:
```suggestion
  "gWRF8m": "Trigger conditions",
```

File: Localize/lang/strings.json [hunk 47]
```diff
@@ -2199,6 +2191,7 @@
   "gMhc5o": "Required. The string to combine into a single string.",
   "gQt/0f": "Conversion functions",
   "gRUmiA": "Info",
+  "gRyFIu": "Copy value of {label} to clipboard",
   "gTXyUo": "Time Zone",
   "gUp23R": "The name already exists or is invalid, please update it before proceeding.",
```
Comment (ecfan) [inline Localize/lang/strings.json:2196]: "Please" is unnecessary.

```suggestion
  "gUp23R": "The name already exists or is invalid. Update the name before you continue.",
```

File: Localize/lang/strings.json [hunk 48]
```diff
@@ -2199,6 +2191,7 @@
   "gMhc5o": "Required. The string to combine into a single string.",
   "gQt/0f": "Conversion functions",
   "gRUmiA": "Info",
+  "gRyFIu": "Copy value of {label} to clipboard",
```
Comment (ecfan) [inline Localize/lang/strings.json:2194]:
```suggestion
  "gRyFIu": "Copy the value of '{label}' to the clipboard",
```

File: Localize/lang/strings.json (hunk 38 above)
Comment (ccastrotrejo) [reply Localize/lang/strings.json:774]: Yes

File: Localize/lang/strings.json [hunk 49]
```diff
@@ -2199,6 +2191,7 @@
   "gMhc5o": "Required. The string to combine into a single string.",
```
[the remark this answers is not in the document]
Comment (ccastrotrejo) [reply Localize/lang/strings.json:2191]: I think it just a single string

File: .vscode/launch.json
[the remark this answers is not in the document]
Comment (ecfan) [reply .vscode/launch.json:1]: Please update for consistency? Thanks!

File: Localize/lang/strings.json (hunk 49 above)
[the remark this answers is not in the document]
Comment (ecfan) [reply Localize/lang/strings.json:2191]: So, does the description exist for multiple string parameters to combine into a single string? Or, is there only one parameter where you enter the strings that you want to combine?

[the remark this answers is not in the document]
Comment (ccastrotrejo) [reply Localize/lang/strings.json:2187]: yep its the second one
Screenshot 2023-06-08 at 10 19 23 AM

[the remark this answers is not in the document]
Comment (ecfan) [reply Localize/lang/strings.json:2187]: Thanks, I clarified so that the text is clearer.

File: Localize/lang/strings.json [hunk 50]
```diff
@@ -264,12 +257,7 @@
   "ALImBL": "(UTC+00:00) Monrovia, Reykjavik",
   "ANbEeQ": "Required. The name of the action whose values you want.",
```
Comment (ecfan) [inline Localize/lang/strings.json:246]: An action isn't a "who":

```suggestion
  "ANbEeQ": "Required. The name of the action that has the values you want.",
```

**Review by refortie — APPROVED** [review]

**Pull request merged**